libclc/CMakeLists.txt

```diff
@@ -244,6 +244,16 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
 		endif()
 		message( "    DEVICE: ${d} ( ${${d}_aliases} )" )
 
+		# FIXME: this is a hack, remove once we can use sycldevice in the triple
```

Comment:
Is this related to the issue #1814? If so, it might be useful to add a link to the issue, so later we can easily check if this hack can be removed.

Reply:
It is, I've updated the comment